apps/site/pages/components/molecules/discount-badge.mdx

```diff
@@ -79,17 +79,21 @@ A custom `Badge` that display product's discounts.
 
 Import the component from [@faststore/ui](/../components)
 
-```tsx
+```tsx copy
 import { DiscountBadge } from '@faststore/ui'
 ```
 
-Import Styles
+### Import Styles into your FastStore project
 
-```tsx
-import '@faststore/ui/src/components/atoms/Badge/styles.scss'
-import '@faststore/ui/src/components/molecules/DiscountBadge/styles.scss'
+To apply the styles of this component in your FastStore project, import the following into your stylesheet:
```

Comment:
@Mariana-Caetano I think we can keep the import files here for each component + the link to the guide. It's easier to copy and paste the needed styles.